platform/mbed_application.c

```diff
@@ -89,10 +89,16 @@ void mbed_start_application(uintptr_t address)
 
 static void powerdown_nvic()
 {
-    int isr_groups_32;
     int i;
+#if defined(__CORTEX_M0PLUS)
+    NVIC->ICER[0] = 0xFFFFFFFF;
+    NVIC->ICPR[0] = 0xFFFFFFFF;
+    for (i = 0; i < 8; i++) {
+        NVIC->IP[i] = 0x00000000;
+    }
+#else
```

Comment:
As an alternative to having this as a separate block could you set `isr_groups_32 = 1` for m0+ and reuse the code below?

Reply:
Good point. I'll do that.

Reply:
Fixed in latest commit 👍